lib/pure/bitopts.nim

```diff
@@ -0,0 +1,283 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2017 Nim Authors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## This module implements a series of low level methods for bit manipulation.
+## By default, this module use fairly optimized procs written in pure nim.
+##
+## To improve performance, you can use the `builtinBitOpts` flag. This unlock
```

Comment:
Will switch to compiler intrinsics by default.
Actually, I think it already works for compile time.
see: static block at the end of  tests/stdlib/tbitopts.nim
but I might be wrong